MuleSoft/Accessibility/InaccessibleWords.yml

```diff
@@ -22,4 +22,7 @@ tokens:
   - upper
   - lower
   - beside
-  - corner
\ No newline at end of file
+  - corner
+  exceptions:
```

Comment:
this has to be outdented or vale will fail due to syntax error:

`  exceptions:`
to
`exceptions:`